packages/message-encryption/src/symmetric.ts

```diff
@@ -197,10 +206,13 @@ class Decoder extends DecoderV0 implements IDecoder<DecodedMessage> {
 export function createDecoder(
   contentTopic: string,
   symKey: Uint8Array,
-  pubsubTopicShardInfo?: SingleShardInfo
+  pubsubTopicShardInfo?: SingleShardInfo,
+  autosharding = false
```

Comment:
If we decide to move forward with adding this parameter - other `create` functions should follow too

Reply:
I forgot to replace this

Reply:
fixed in latest commit